ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPartyManagementService.scala

```diff
@@ -362,6 +367,22 @@ private[apiserver] final class ApiPartyManagementService private (
     }
   }
 
+  // Here we check if party exists and actually belongs to the requested Identity Provider
+  private def partyBelongsToIDPOrError(
```

Comment:
```suggestion
  private def verifyPartyIsNonExistentOrInIdp(
```
?
Current name suggests that a non-existent party, which obviously doesn't belong to any idp, would case an error